variables.tf

```diff
@@ -3,6 +3,16 @@ variable "cluster_id" {
   type        = string
 }
 
+variable "ecrpublic_username" {
+  description = "EKS Cluster Id"
```

Comment:
lets update the descriptions - just copy paste the text from the resource https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/ecrpublic_authorization_token#password